assets/blocks/course-outline/module-block/edit.js

```diff
@@ -29,24 +28,44 @@ import { useInsertLessonBlock } from './use-insert-lesson-block';
  * @param {string}   props.attributes.blockStyle  Selected block style.
  * @param {Object}   props.mainColor              Header main color.
  * @param {Object}   props.textColor              Header text color.
+ * @param {Object}   props.borderColor            Border color.
  * @param {Function} props.setAttributes          Block set attributes function.
  */
 export const EditModuleBlock = ( props ) => {
 	const {
 		clientId,
 		className,
-		attributes: { title, description },
+		attributes: { title, description, bordered, borderColorValue },
 		mainColor,
 		textColor,
 		setAttributes,
-		blockStyle,
+		name,
 	} = props;
 	const {
-		outlineAttributes: { collapsibleModules },
+		outlineAttributes: { collapsibleModules, moduleBorder },
 	} = useContext( OutlineAttributesContext ) || { outlineAttributes: {} };
 
 	useInsertLessonBlock( props );
 
+	const [ outlineClass, setOutlineClass ] = useState( null );
+
+	// setOutlineClass is called when there is an update in course style class only.
+	// eslint-disable-next-line react-hooks/exhaustive-deps
+	useEffect( () => {
+		applyParentStyle(
```

Comment:
I'm wondering if handle this in the Course block would make more sense, similar to the `bordered`. WDYT? Maybe it could also prevent to run the `applyParentStyle` every module render.

Reply:
Good idea! It didn't make any sense to be in the module block. Probably it was left there after trying different approaches in the beggining. I also took the chance to clean it up a bit. cfa5d5be4f0004c6091738a942a7edcf0372bc16